backend/api/router/router.go

```diff
@@ -159,6 +159,48 @@ func SetupRoutes(i *do.Injector, app *gin.Engine) error {
 		return err
 	}
 
+	// likePost /posts/{postId}/like POST
+	app.POST("/posts/:postid/like", postController.LikePost)
+	likePostOpe, _ := appDoc.NewOperationContext(http.MethodPost, "posts/{postId}/like")
+	likePostOpe.SetID("likePost")
+	likePostOpe.AddReqStructure(new(schema.LikePostRequest))
+	likePostOpe.SetSummary("対象の投稿をlikeする")
+	likePostOpe.SetTags("post")
+	likePostOpe.AddRespStructure(new(schema.MutationSchema), openapi.WithHTTPStatus(http.StatusCreated))
+	likePostOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusBadRequest))
+	likePostOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusForbidden))
+	likePostOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusInternalServerError))
+	if err := appDoc.AddOperation(likePostOpe); err != nil {
+		return err
+	}
+	// getlikeRecord /posts/{postId}/getLikes POST
+	app.GET("/posts/:postid/getLikes", postController.GetLikeRecords)
+	getLikeRecordsOpe, _ := appDoc.NewOperationContext(http.MethodPost, "posts/{postId}/getLikes")
+	getLikeRecordsOpe.SetID("likePost")
+	getLikeRecordsOpe.AddReqStructure(new(schema.GetLikesRequest))
+	getLikeRecordsOpe.SetSummary("対象の投稿のlike数を取得する")
+	getLikeRecordsOpe.SetTags("post")
+	getLikeRecordsOpe.AddRespStructure(new(schema.LikeRecordResponse), openapi.WithHTTPStatus(http.StatusOK))
+	getLikeRecordsOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusBadRequest))
+	getLikeRecordsOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusForbidden))
+	getLikeRecordsOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusInternalServerError))
+	if err := appDoc.AddOperation(getLikeRecordsOpe); err != nil {
+		return err
+	}
+	// updatelikeRecord /posts/updateLikes POST
+	app.POST("/posts/updateLikes", postController.UpdateLikeRecords)
+	updateLikeRecordsOpe, _ := appDoc.NewOperationContext(http.MethodPost, "posts/updateLikes")
```

Comment:
`POST /posts/like/update`のほうがいいかも
```suggestion
	// updatelikeRecord /posts/like/update POST
	app.POST("/posts/like/update", postController.UpdateLikeRecords)
	updateLikeRecordsOpe, _ := appDoc.NewOperationContext(http.MethodPost, "posts/like/update")
```